dev/com.ibm.websphere.appserver.features/visibility/public/jaxws-2.3/com.ibm.websphere.appserver.jaxws-2.3.feature

```diff
@@ -6,35 +6,16 @@ IBM-App-ForceRestart: uninstall, \
  install
 IBM-ShortName: jaxws-2.3
 IBM-SPI-Package: com.ibm.wsspi.webservices.handler
-IBM-API-Package: \
- javax.jws.soap; type="spec"; require-java:="9", \
- javax.wsdl; type="spec", \
- javax.wsdl.extensions; type="spec", \
- javax.wsdl.extensions.http; type="spec", \
- javax.wsdl.extensions.mime; type="spec", \
- javax.wsdl.extensions.schema; type="spec", \
- javax.wsdl.extensions.soap; type="spec", \
- javax.wsdl.extensions.soap12; type="spec", \
- javax.wsdl.factory; type="spec", \
- javax.wsdl.xml; type="spec", \
- javax.xml.soap; type="spec"; require-java:="9", \
- javax.xml.ws; type="spec", \
- javax.xml.ws.handler; type="spec", \
- javax.xml.ws.handler.soap; type="spec", \
- javax.xml.ws.http; type="spec", \
- javax.xml.ws.soap; type="spec", \
- javax.xml.ws.spi; type="spec", \
- javax.xml.ws.spi.http; type="spec", \
- javax.xml.ws.wsaddressing; type="spec", \
+IBM-API-Package: 
  org.apache.cxf.binding.soap.wsdl.extensions;type="internal", \
- org.apache.cxf.databinding;type="internal", \
- javax.jws; type="spec"; require-java:="9"
+ org.apache.cxf.databinding;type="internal"
 Subsystem-Name: Java Web Services 2.3
--features=\
- com.ibm.websphere.appserver.jaxb-2.3, \
- com.ibm.websphere.appserver.internal.jaxws-2.3, \
- com.ibm.websphere.appserver.eeCompatible-8.0
+-features= \
+ com.ibm.websphere.appserver.globalhandler-1.0, \
+ com.ibm.websphere.appserver.jaxws.common-2.3, \
+ com.ibm.websphere.appserver.servlet-4.0
```

Comment:
If it does become 2.2, we cannot depend on servlet of course like we talked about.

Reply:
Yep, absolutely true I elected to keep it for now, as it will require making updates to the fat repeat rules.